### PR TITLE
improvements to webscrapper

### PR DESCRIPTION
**Pull request by kiosia**
tested locally on chrome only

tried to test on android, but it was late

kthxbai

### CODE REVIEW

**Comment by kiosia** [comment]
@aquintiliano @cristiadu façam seus shows

**Review by aquintiliano — APPROVED** [review]

File: api/integrations/fifaIndexWebScrapper.js [hunk 1]
```diff
@@ -3,12 +3,12 @@ const cheerioDOMLoader = require('cheerio');
 const fs = require('fs');
 
 const BASE_URL = "https://www.fifaindex.com"
-const BASE_QUERY = 'SELECT * FROM htmlstring WHERE url="https://www.fifaindex.com/teams/{0}/?type={1}"';
+const BASE_QUERY = 'SELECT * FROM htmlstring WHERE url="https://www.fifaindex.com/teams/fifa17_74/{0}/?type={1}"';
```
Comment (cristiadu) [inline api/integrations/fifaIndexWebScrapper.js]: No hardcoded versions here, my friend.

**Review by cristiadu — CHANGES_REQUESTED** [review]
One small fix.

**Review by cristiadu — APPROVED** [review]
Hardcoded string needs to be removed later on

**Pull request merged**